src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -100,8 +101,8 @@ void QueueNotifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const st
     spathList = "";
   }
 
-  ci.outFormat = format;
-  std::string payload = ncr->render(&ci, NotifyContext, "");
+  ci.outFormat = JSON;
```

Comment:
If mixed, the values would be prefixed, like jsonValues, jsonKeyvalues, etc.
It's ok and it removes the use of an 'extra' parameter.
Using two parameters is cleaner, no doubt.
To me, both ways are ok